### PR TITLE
nsm v1.2 uplift

### PR DESCRIPTION
**Pull request by zolug**
related to:
https://github.com/Nordix/Meridio/issues/140

### CODE REVIEW

File: deployment/ipam.yaml
Comment (mandydydy) [inline deployment/ipam.yaml:20]: the spiffe labels are implemented in https://github.com/Nordix/Meridio-Operator/pull/55, Lionel is waiting for the PR in Meridio to be merged first. Maybe you can rebase on that after that is merged?

File: controllers/attractor/lb-fe.go
Comment (mandydydy) [inline controllers/attractor/lb-fe.go:79]: I don't see these xxx_LOG_LEVEL added in the yaml files. Is it intended to use the default values?

File: controllers/attractor/lb-fe.go
Comment (mandydydy) [inline controllers/attractor/lb-fe.go:141]: I don't see NFE_LOG_LEVEL in Meridio nsm uplift PR. Shouldn't it exist in both PRs?

File: controllers/trench/nsp.go
Comment (mandydydy) [inline controllers/trench/nsp.go:48]: same comment with NSP_LOG_LEVEL

File: deployment/ipam.yaml
Comment (zolug) [reply deployment/ipam.yaml:20]: I don't see a point for doing the spiffe label thing separately. The PR was implemented and tested with labels included, so wouldn't prefer changing that just for the sake of a small PR.

File: controllers/attractor/lb-fe.go
Comment (zolug) [reply controllers/attractor/lb-fe.go:79]: Yes, the default for these *LOG_LEVEL env values is controlled through the respective go codes.
I just wanted to add support to the code for including them from the templates if present. But if you think it'd be better to add them explicitly to the templates, then I can do that.

**Review by mandydydy — APPROVED** [review]

**Pull request merged**
